ank/src/cli_commands.rs

```diff
@@ -42,6 +42,7 @@ use url::Url;
 use crate::{cli::OutputFormat, output_and_error, output_debug};
 
 const BUFFER_SIZE: usize = 20;
+const WAIT_TIME_MS: u128 = 3000;
```

Comment:
We could use Duration here:
```suggestion
const WAIT_TIME_MS: Duration = Duration::from_millis(3000);
```